NextcloudTalk/BaseChatViewController.swift

```diff
@@ -1491,6 +1518,77 @@ import QuickLook
     func hideVoiceMessageRecordingView() {
         self.voiceMessageRecordingView?.isHidden = true
     }
+    
+    // MARK: - Expanded voice message recording
+
+    func showExpandedVoiceMessageRecordingView(offset: Int) {
+        
+        let expandedView = ExpandedVoiceMessageRecordingView(
+            deleteFunc: handleDelete, sendFunc: handleSend, recordFunc: handleRecord(isRecording:), timeElapsed: offset
+        )
+        
+        self.expandedUIHostingController = UIHostingController(rootView: expandedView)
+        self.textInputbar.addSubview(self.expandedUIHostingController!.view)
+        
+        guard let expandedVoiceMessageRecordingView = self.expandedUIHostingController!.view else { return }
+        expandedVoiceMessageRecordingView.translatesAutoresizingMaskIntoConstraints = false
+        
+        
+        let views = [
+            "expandedVoiceMessageRecordingView": expandedVoiceMessageRecordingView
+        ]
+        
+        self.textInputbar.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[expandedVoiceMessageRecordingView]|", metrics: nil, views: views))
+        self.textInputbar.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|[expandedVoiceMessageRecordingView]|", metrics: nil, views: views))
+    
+    }
```

Comment:
```suggestion
    func showExpandedVoiceMessageRecordingView(offset: Int) {
        let expandedView = ExpandedVoiceMessageRecordingView(
            deleteFunc: handleDelete, sendFunc: handleSend, recordFunc: handleRecord(isRecording:), timeElapsed: offset
        )

        let hostingController = UIHostingController(rootView: expandedView)
        guard let expandedVoiceMessageRecordingView = hostingController.view else { return }

        self.expandedUIHostingController = hostingController
        self.textInputbar.addSubview(expandedVoiceMessageRecordingView)

        expandedVoiceMessageRecordingView.translatesAutoresizingMaskIntoConstraints = false

        let views = [
            "expandedVoiceMessageRecordingView": expandedVoiceMessageRecordingView
        ]

        self.textInputbar.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[expandedVoiceMessageRecordingView]|", metrics: nil, views: views))
        self.textInputbar.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|[expandedVoiceMessageRecordingView]|", metrics: nil, views: views))
    }
```

This way you don't have to force-unwrap anything.